api/swagger-spec/swagger.yml

```diff
@@ -275,6 +301,13 @@ definitions:
         type: array
         items:
           $ref: "#/definitions/componentVersion"
+  doctorInfo:
+    type: object
+    required:
+      - cluster
+    properties:
+      cluster:
+        $ref: "#/definitions/cluster"
```

Comment:
That's fair to exclude the report id from the data model. I'd prefer to keep it in the route endpoint so our GET route looks the same as our POST route and we can have CRUD nirvana.